rpc-client-interface.go

```diff
@@ -63,4 +63,9 @@ type NetLocker interface {
 
 	// Return this lock server service endpoint on which the server runs.
 	ServiceEndpoint() string
+
+	// Init attempts to initialize the underlying remote connection.
+	// if already initialized closes the connection and re-connects,
+	// if not initializes a new rpc connection.
+	Init() error
```

Comment:
Right Init() should be just used inside no need to expose it outside of NetLocker implementation.